scss/main.scss

```diff
@@ -1 +1,88 @@
 @use 'vars';
+
+// Track Circle Class
+@mixin circle($color, $size) {
+  background-color: $color;
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+  display: inline-block;
+}
+// Track Box Class
+@mixin box($color) {
+  border: 4px solid $color;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+// Track Circles
+.beginner-circle {
+  @include circle(#82ECEC, 183px);
+}
+
+.general-circle {
+  @include circle(#A88AA8, 183px);
+}
+
+.research-circle {
+  @include circle(#EA668E, 183px);
+}
+
+// Footer Circles
+.blue-circle {
+  @include circle(#82ECEC, 51px);
+}
+
+.purple-circle {
+  @include circle(#A88AA8, 51px);
+}
+
+.pink-circle {
+  @include circle(#EA668E, 51px);
+}
+
+// Track Boxes
+.beginner {
+  @include box(#82ECEC);
+}
+
+.general {
+  @include box(#A88AA8);
+}
+
+.research {
+  @include box(#EA668E);
+}
+
+.intro {
+  padding: 64px 32px;
+  background-color: #DED2E6;
+}
+
+.intro-header {
+  padding-top: 44px;
+  padding-bottom: 1%;
+}
+
+.tracks {
+  padding: 32px;
+}
+
+.track-header {
+  padding-bottom: 1%;
+}
+
+.rectangle {
+  display: block;
+  height: 295px;
+  width: 485px;
+  top: 10px;
+  padding-right: 30px;
+}
+
+.outer-intro-layer {
+  background-color: white;
+  padding-left: 32px;
+  padding-right: 32px;
```

Comment:
nit: imo this "padding-right" isnt needed. same goes for "background-color" since the container of this intro section already has a white background.